main/main.go

```diff
@@ -0,0 +1,298 @@
+package main
```

Comment:
You want this file under the `cmd/sops` directory, not `main`. At the moment, `go get go.mozilla.org/sops/main` would install a program called `main`.